config/maximal-config-example.toml

```diff
@@ -33,6 +33,13 @@ port = 7300
 # that is used by the `indexer-agent`. It is expected that `indexer-agent` will create
 # the necessary tables.
 postgres_url = "postgres://postgres@postgres:5432/postgres"
+# You can also use the following fields to specify the connection details separately.
+# either use `postgres_url` or the following fields.
+postgres_host = "postgres-host"
+postgres_port = 5432
+postgres_user = "user"
+postgres_password = "pwd"
+postgress_db = "postgres"
```

Comment:
same here

Reply:
you could use one style in the minimal config, and another in the maximal config so that means we are testing both. Don't forget to leave the comment "as alternative you can use this other one"

Reply:
I prefer both having same style to make the whole thing cohesive

Reply:
"as alternative you can use this other one", what did yopu meant by this?
There is already a comment 
```
# You can also use the following fields to specify the connection details separately.
# either use `postgres_url` or the following fields.
```